doc/usage/tutorial.md

```diff
@@ -45,10 +33,68 @@ NAME        	REVISION	UPDATED                 	STATUS  	CHART                	NA
 ```
 
-NOTE: `kube-batch` need to collect cluster information(such as Pod, Node, CRD, etc) for scheduling, so the service account used by the deployment must have permission to access those cluster resources, otherwise, `kube-batch` will fail to startup. For users who are not familiar with Kubernetes RBAC, please copy the example/role.yaml into `$GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch/templates/` and reinstall batch.
+## Customize kube-batch
+
+Please check following configurable fields and customize kube-batch based on your needs.
+
+### kube-batch configuraions
+| configs | default value | Descriptions |
+| ----------- | ----------- | ------ | ----------- |
+| master | "" | The address of the Kubernetes API server (overrides any value in kubeconfig) |
+| kubeconfig | "" | Path to kubeconfig file with authorization and master location information. If you run kube-batch outside cluster or for debug purpose, this is very useful |
+| scheduler-name | kube-batch | kube-batch will handle pods with the scheduler-name |
+| scheduler-conf | "" | The absolute path of scheduler configuration file |
+| default-queue | default | The default queue name of the job |
+| schedule-period | Seconds | The period between each scheduling cycle |
+| leader-elect | false | Start a leader election client and gain leadership before executing the main loop. Enable this when running replicated kube-batch for high availability |
+| lock-object-namespace | Define the namespace of the lock object |
+| listen-address | :8080 | The address to listen on for HTTP requests. Metrics will be exposed at this port |
+
+
+### scheduler configuration
+
+By default, scheduler will read following default configuraion as `scheduler-conf`.
+
+```yaml
+actions: "allocate, backfill"
+tiers:
+- plugins:
+  - name: priority
+  - name: gang
+- plugins:
+  - name: drf
+  - name: predicates
+  - name: proportion
+  - name: nodeorder
+```
+
+
+The `actions` is a list of actions that will be executed by kube-batch in order, although the "order" maybe incurrect; the kube-batch do not enforce that. In above example, `allocate, backfill` will be executed in order by kube-batch.
 
-### (3) Create a Job
+The `plugins` is a list of plugins that will be used by related actions, e.g. allocate. It includes several tiers of plugin list by names; if it fix plugins in high priority tier, the action will not go through the plugins in lower priority tiers. In each tier, it's considered passed if all plugins are fitted in `plugins.names`.
 
+multiple tiers for plugins is introduced into kube-batch, high priority jobs will take all resources it need; if priority equal, shares resource by DRF.
+
+Currently, kube-batch support
+- actions
+  - allocate
+  - backfill
+  - preempt
+  - reclaim
+- plugins
+  - conformance
+  - drf
+  - gang
+  - nodeorder
+  - predicates
+  - priority
+  - proportion
+
+
+## kube-batch Features
```

Comment:
Change sequential steps to features. I think the straightforward way is to explain these plugins and give example?

Reply:
Anything else we can improve tutorial?